config/application.properties

```diff
@@ -1,5 +1,6 @@
 #Bridge related settings
 bridge.id=my-bridge
+bridge.tracing=jaeger
```

Comment:
Should this be enabled by default?

Reply:
What do you mean? I am kinf of reproducing what we have in the operator for enabling tracing which is not enabled by default right? You have this `tracing` property to set in the custom resource.

Reply:
Yeah. ut the user has to add it there. Where as here you seem to add it as default into the configs in the repository. Or did I misunderstood how this file is used?

Reply:
Now I got what you meant. You were saying that having it year will enable tracing by default which we don't want :) I will remove it